woodwork/tests/test_utils.py

```diff
@@ -436,6 +438,39 @@ def test_is_null_latlong():
     assert not _is_null_latlong(False)
 
 
+def test_is_valid_latlong_value():
+    values = [
+        (1.0, 2.0),
+        (np.nan, np.nan),
+        [1.0, 2.0],
+        np.nan,
```

Comment:
Based off of these lines in `_reformat_to_latlong`, should this be a valid latlong?

```python
        # (np.nan, np.nan) should be counted as a single null value
        if pd.isnull(latitude) and pd.isnull(longitude):
            return np.nan
```

Reply:
Ah, yes. I was thinking `(np.nan, np.nan)` was the valid null value, but it's just `np.nan`.So I think our valid data should be:
- a tuple with two floats, provided they are not both `np.nan`
- a single `np.nan` value

Is that consistent with your thoughts, @tamargrey?

Reply:
yep!